website/src/pages/guides/dev-onboarding-how-is-it-unique/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../../components/Layout'
+import { useHubSpot } from '../../../hooks/hubSpot'
+import styles from './index.module.scss'
+
+const DevOnboardingUnique: FunctionComponent<PageProps> = props => {
+    useHubSpot({
+        portalId: '2762526',
+        formId: '25249bac-6544-4f50-9fc0-559745de5334',
+        targetId: 'form',
+        chiliPiper: false,
+        onFormSubmitted: () => window.open('/guides/dev-onboarding-unique/sg-dev-onboarding-what-makes-it-unique.pdf'),
```

Comment:
Missing assets

Reply:
These have been added in bba7b52

Reply:
Thanks, @katjuell! It looks like the PDF I placed there was incorrect. Apologies for the mixup. The correct PDF is there now, entitled `sg-dev-onboarding-what-makes-it-unique.pdf`, so the path should remain the same. 0af139b